src/app/inventory/d2-stores.ts

```diff
@@ -272,15 +275,19 @@ function makeD2StoresService(): D2StoreServiceType {
     store.progression = progressions ? { progressions: Object.values(progressions) } : null;
 
     // We work around the weird account-wide buckets by assigning them to the current character
-    let items = characterInventory.concat(Object.values(characterEquipment));
+    const items = characterInventory;
```

Comment:
Hmm, now that I look at this it seems like a (relatively benign) behavior change where we're mutating `characterInventory` rather than creating a new array.

Reply:
Fixed, better safe than sorry since it looks like this mutation may have unexpected side effects